### ISSUE TITLE
No obvious way to name autocomplete input

### ISSUE THREAD

**Issue by six8** [ninja/ui#50]
You can't specify the name for the input field with the normal autocomplete options.

It would be nice to do:

```
$autocomplete = $.ninja.autocomplete({name: 'q'})
```

However this just puts the name on the wrapping span.

Currently you have to do the following which is neither obvious or convenient:

```
$autocomplete.find('input').attr('name', 'q')
```

**Comment by uipoet** [comment]
Good point. I think the best solution will be to accept an array of attributes as an option and pass them all through to the input, like how we pass through css and html in other areas:

```
$autocomplete = $.ninja.autocomplete({
  attr: {
    name: 'q'
  }
}
```
